bills-app/src/main/java/com/google/sps/servlets/TranslationServlet.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.cloud.translate.*;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/translate")
+public class TranslationServlet extends HttpServlet {
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException { 
+    // Get the request parameters.
+    String originalText = "This text will be translated."; //request.getParameter("text");
+    String languageCode = "es"; //request.getParameter("languageCode");
```

Comment:
I think this could be converted into a constant and have it be outside the doPost() function.

Reply:
I agree! You can add a constant DEFAULT_TRANSLATION_LANGUAGE for readability, and this makes it easier in the future if we choose to change the translation language (instead of having to read through the code to figure out where it is if it gets more complext).